src/main/kotlin/io/billie/organisations/orders/service/OrderService.kt

```diff
@@ -0,0 +1,42 @@
+package io.billie.organisations.orders.service
+
+import io.billie.organisations.orders.data.OrderNotFoundException
+import io.billie.organisations.orders.data.OrderRepository
+import io.billie.organisations.orders.data.ShipmentNotificationFailure
+import io.billie.organisations.orders.domain.Order
+import io.billie.organisations.orders.domain.OrderId
+import io.billie.organisations.orders.domain.Shipment
+import io.billie.organisations.orders.domain.ShipmentId
+import io.billie.organisations.viewmodel.Entity
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+/**
+ * Offers services to fetch or create order per organisation
+ */
+@Service
+class OrderService(val repository: OrderRepository) {
+
+    @Transactional(readOnly = true)
```

Comment:
Order service which offers method to create, fetch orders and notifying shipment for an order